suites/quincy/rbd/tier-3_rbd_persistent_write_back_cache.yaml

```diff
@@ -461,3 +461,27 @@ tests:
       module: test_performance_pwl_cache.py
       name: Performance comparison with or without pwl cache
       polarion-id: CEPH-83574711
+
+  - test:
+      abort-on-fail: true
+      config:
+        level: client                        # PWL at client
+        cache_file_size: 1073741824          # 1 GB
+        rbd_persistent_cache_mode: ssd       # "ssd" or "rwl" on pmem device
+        client: node5
+        drive: /dev/nvme0n1
+        cleanup: true
+        rep-pool-only: True
+        rep_pool_config:
+          pool: pool1
+          image: image1
+          size: 10G
+        fio:
+          image_name: image2
```

Comment:
Is fio being run on a different image and persistent cache setup on a different image?

Reply:
sorry renamed